src/SprykerEco/Zed/Computop/Business/Api/Mapper/PrePlace/AbstractPrePlaceMapper.php

```diff
@@ -129,6 +138,12 @@ protected function getComputopPaymentTransfer(QuoteTransfer $quoteTransfer, Comp
             $this->computopService->getMacEncryptedValue($computopPaymentTransfer)
         );
 
+        $paymentEntity = $this->queryContainer
```

Comment:
Please extract this to a separate method or even model, since in Mapper it's not nice to read from DB.